<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>足球球員管理系統</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelName">
        <property name="text">
         <string>球員姓名:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPosition">
        <property name="text">
         <string>位置:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxPosition">
        <item>
         <property name="text">
          <string>前鋒</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>中場</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>後衛</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>守門員</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelAge">
        <property name="text">
         <string>年齡:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBoxAge">
        <property name="minimum">
         <number>15</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonAdd">
      <property name="text">
       <string>新增球員</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidgetPlayers">
      <property name="columnCount">
       <number>3</number>
      </property>
      <column>
       <property name="text">
        <string>姓名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>位置</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>年齡</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxPractice">
      <property name="title">
       <string>進球練習記錄</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="labelPracticePlayer">
           <property name="text">
            <string>選擇球員:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBoxPracticePlayer"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelDistance">
           <property name="text">
            <string>距離(碼):</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBoxDistance">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>50</number>
           </property>
           <property name="value">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelTotalAttempts">
           <property name="text">
            <string>計劃嘗試次數:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="spinBoxTotalAttempts">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonStartPractice">
         <property name="text">
          <string>開始練習</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelPracticeInfo">
         <property name="text">
          <string>尚未開始練習</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButtonRecordGoal">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>記錄成功進球</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonRecordMiss">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>記錄未進球</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonEndPractice">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>結束練習</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxStats">
      <property name="title">
       <string>數據分析</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButtonShowStats">
         <property name="text">
          <string>顯示統計數據</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidgetStats">
         <property name="columnCount">
          <number>6</number>
         </property>
         <column>
          <property name="text">
           <string>球員</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>距離(碼)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>總嘗試</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>成功進球</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>成功率(%)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>時間(秒)</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
